File: seed/pydantic-v2/trace/resources/v_2/resources/v_3/resources/problem/types/test_case_with_actual_result_implementation.py
from pydantic import BaseModel
from resources.v_2.resources.v_3.resources.problem.types.non_void_function_definition import NonVoidFunctionDefinition
from resources.v_2.resources.v_3.resources.problem.types.assert_correctness_check import AssertCorrectnessCheck
from dt import datetime
from core.datetime_utils import serialize_datetime
class TestCaseWithActualResultImplementation(BaseModel):
    get_actual_result: NonVoidFunctionDefinition = 
    assert_correctness_check: AssertCorrectnessCheck = 
    class Config:
        frozen = True
        smart_union = True
        json_encoders = {datetime: serialize_datetime}

File: seed/ruby-sdk-v2/trace/lib/seed/v_2/v_3/problem/test_case_with_actual_result_implementation.rb

module Seed
    module Types
        class TestCaseWithActualResultImplementation < Internal::Types::Model
            field :get_actual_result, Seed::v_2::v_3::problem::NonVoidFunctionDefinition, optional: false, nullable: false
            field :assert_correctness_check, Seed::v_2::v_3::problem::AssertCorrectnessCheck, optional: false, nullable: false

    end
end
